hw/top_earlgrey/ip/flash_ctrl/data/autogen/flash_ctrl_sec_cm_testplan.hjson

```diff
@@ -293,16 +293,5 @@
       stage: V2S
       tests: ["flash_ctrl_sec_cm"]
     }
-    {
```

Comment:
@jdonjdon 
The sec_cm files are auto-generated once and then meant to be manually editted.
This part gets a bit confusing because the block is templated, but @jdonjdon can speak more to it.

Reply:
@jeoongp @tjaychen Sorry, I didn't know this can cause lint error.
Fixed #15643

Reply:
Thanks @jdonjdon!